src/settings-renderer/components/dialogs/MenuThemesDialog.tsx

```diff
@@ -115,6 +115,7 @@ export default function MenuThemesDialog() {
             gap: 10,
             justifyContent: 'space-between',
           }}>
+          <Scrollbox maxHeight={'min(20vh, 400px)'} width="100%">
```

Comment:
If you set the left padding to zero, the alignment looks a bit better, I think.


```suggestion
          <Scrollbox maxHeight={'min(20vh, 400px)'} width="100%" paddingLeft={0}>
```